### ISSUE TITLE
commandlinerunner-log4j2 fails with GraalVM 22.1

### ISSUE THREAD

**Issue by mhalbritter** [spring-projects-experimental/spring-native#1547]
```
openjdk version "17.0.3" 2022-04-19
OpenJDK Runtime Environment GraalVM CE 22.1.0-dev (build 17.0.3+4-jvmci-22.1-b03)
OpenJDK 64-Bit Server VM GraalVM CE 22.1.0-dev (build 17.0.3+4-jvmci-22.1-b03, mixed mode, sharing)
```

```
========================================================================================================================
GraalVM Native Image: Generating 'commandlinerunner-log4j2' (executable)...
========================================================================================================================

[1/7] Initializing...                                                                                    (0,0s @ 0,07GB)
Error: ImageSingletons do not contain key com.oracle.svm.hosted.LinkAtBuildTimeSupport
Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
------------------------------------------------------------------------------------------------------------------------
                         0,1s (6,4% of total time) in 8 GCs | Peak RSS: 0,59GB | CPU load: 5,14
========================================================================================================================
Failed generating 'commandlinerunner-log4j2' after 1,4s.
```

Looks like https://github.com/spring-projects-experimental/spring-native/issues/1546

**Comment by dr-eme** [comment]
I had a similar issue with a slightly modified commandlinerunner-gradle sample, and opened the following ticket with graal:
https://github.com/oracle/graal/issues/4424
In principle, the issue was solved, but maybe not yet merged in the latest dev build.
Maybe, this applies also to https://github.com/spring-projects-experimental/spring-native/issues/1546

**Comment by olpaw** [comment]
This is already fixed on master and the 22.1 release branch. See
https://github.com/oracle/graal/pull/4427
https://github.com/oracle/graal/commits/release/graal-vm/22.1 (see commit 4f2702d80b79a965eaf44a64ac7c6df744dbb01a)

**Comment by mhalbritter** [comment]
Works with graal built from https://github.com/oracle/graal/commit/3eb1d111ab25565acbbc015128a2c787656e2675
